### DISCUSSION TITLE
[@types/node] "node:test" suite function is defined but not exported

### DISCUSSION THREAD

**Discussion by fvsch** [DefinitelyTyped/DefinitelyTyped#70090] [Issues with a @types package] [answered]
- Node feature: https://github.com/nodejs/node/pull/52127
- Shipped in 20.13.0: https://github.com/nodejs/node/pull/52793
- Types added in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69508

The `suite` function and namespace are the new name of `describe`, with `describe` being defined as an alias.

The current types are:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a7e0a47435dd0ec1df13a35aba6eb11ace56ab90/types/node/test.d.ts#L157-L186

My issue is that `suite` doesn't seem to be exported at all, resulting in this code showing an error with `@types/node@20.14.11`:

```js
// @ts-check

import { suite } from 'node:test';
//       ^^^^^ Module '"node:test"' has no exported member 'suite'.
//             Did you mean to use 'import suite from "node:test"' instead?
//             ts(2614)
```

… despite the code running correctly in Node (tested in 20.15.1).

Current workaround: import `describe` instead of `suite`.

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment, marked as answer]
@fvsch I'm sure it was already fixed

**Reply by fvsch** [reply]
Looks like it was fixed about a week after I opened this discussion, in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/70125

**Reply by fvsch** [reply]
It's in `@types/node@20.14.13`.

**Comment by alshdavid** [comment]
Also `TestFn`, `TestOptions` are not exported. Useful when creating a wrapper around `test`
